services/app-web/src/pages/SubmissionSummary/SubmissionSummary.tsx

```diff
@@ -161,6 +161,32 @@ export const SubmissionSummary = (): React.ReactElement => {
                 )}
 
                 <SubmissionTypeSummarySection
+                    subHeaderComponent={
+                        isCMSUser ? (
```

Comment:
Since this feature will be incrementally shipped (e.g. we haven't done delete behavior yet, we haven't done edit, we haven't added confirmation banner)I think this work should be wrapped in a feature flag.

Reply:
@haworku I followed the [docs](https://github.com/Enterprise-CMCS/managed-care-review/blob/main/docs/technical-design/launch-darkly-feature-flag-lifecycles.md#steps) for adding a new feature flag, I turned it on for local with `zuko@example.com` and used the flag in this [file](https://github.com/Enterprise-CMCS/managed-care-review/blob/mcr-3547-add-view-mccrs-id/services/app-web/src/pages/SubmissionSummary/SubmissionSummary.tsx#L170) to control displaying the link to add the mccrs id. 

However, I'm seeing that the flag is always false. Is there anything else I need to do to get the flag to turn on?

Reply:
@pearl-truss sorry I missed this glad you brought it to slack